build/linux/installer/conf/telegraf.conf

```diff
@@ -675,7 +675,9 @@
   ## An array of urls to scrape metrics from.
   urls = ["$CADVISOR_METRICS_URL"]
 
-  fieldpass = ["kubelet_running_pod_count","volume_manager_total_volumes", "kubelet_node_config_error", "process_resident_memory_bytes", "process_cpu_seconds_total"]
+  # <= 1.18: metric name is kubelet_running_pod_count
+  # >= 1.19: metric name changed to kubelet_running_pods
+  fieldpass = ["kubelet_running_pod_count","kubelet_running_pods","volume_manager_total_volumes", "kubelet_node_config_error", "process_resident_memory_bytes", "process_cpu_seconds_total"]
```

Comment:
If its either/or (meaning clusters >= 1.19.x will have the new metric, and clusters < 1.19.x will have old metric), we should fix the workbook query to include both the metrics so account for all the cluster versions.

Reply:
Yes, I have created a task for this on the board